cmd/pinniped-proxy/src/main.rs

```diff
@@ -0,0 +1,9 @@
+use structopt::StructOpt;
+
+// Ensure the root crate is aware of the child modules.
+mod cli;
+
+fn main() {
+    let opt = cli::Options::from_args();
+    println!("{:#?}", opt);
```

Comment:
A simple main function to begin with, just does a debug print (`{:#?}`) of the options. More interesting is checking the help for the command (included in the dev docs below, since it won't change).

If you remove the `Debug` from the derive macro in the `cli` module, you'll find it will no longer compile as this line here requires an implementation for displaying the struct in debug.